python/helpers/pydev/pydevd_file_utils.py

```diff
@@ -73,10 +73,29 @@
 
 normcase = os_normcase # May be rebound on set_ide_os
 
+def convert_to_long_pathname(filename):
```

Comment:
I think, we should remove the second `def`, even without inner functions, and check the platform inside the function.
```
def convert_to_long_pathname(filename):
    if os.name != 'nt':
        return filename
    else:
        # the rest of code, without def
```
It's not very important, just minors. I can change it while merging, if you're not sure.